### PR TITLE
[BugFix] Pagination and Serial number column issue is fixed

### PR DESCRIPTION
**Pull request by prateek-yojak**
Pagination is working now and serial number column is also working. You have to write keyName='sno' to get serial number.

### CODE REVIEW

File: src/table/Table.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React from "react";
 import { Checkbox, Pagination, Table } from "semantic-ui-react";
-
+import './table.css';
```
Comment (lakshita-yojak) [inline src/table/Table.tsx:3]: put this at end of external file

File: src/table/Table.tsx [hunk 2]
```diff
@@ -34,13 +38,16 @@ export const TableComponent = ({
...
 	pagination = false,
 	totalPages,
-	currentPage,
+	currentPage = 1,
 	setCurrentPage,
 	url = "",
 	collapsable = false,
 	subCol = [],
 	subRowKey = "",
+	handleRowClick= null
```
Comment (lakshita-yojak) [inline src/table/Table.tsx:47]: add space before =

File: src/table/TableRow.tsx [hunk 3]
```diff
@@ -101,14 +111,23 @@ const TableRow = ({
...
 
 						if (key.length) {
 							const keys = key.split(".");
-							keys.forEach((keyMain: string) => {
-								result = result[keyMain] ? result[keyMain] : "-";
+							keys.forEach((keyMain) => {
+								
+								if(showSerialNumber){
+								result = keyMain in result ? result[keyMain] : index+1;}
+								else if (typeof result[keyMain]==="boolean" || result[keyMain]===0){
```
Comment (lakshita-yojak) [inline src/table/TableRow.tsx:125]: use !result[keyMain] instead of result[keyMain]===0

File: src/table/TableRow.tsx [hunk 4]
```diff
@@ -101,14 +111,23 @@ const TableRow = ({
...
 						let target = "";
 
 						if (key.length) {
 							const keys = key.split(".");
-							keys.forEach((keyMain: string) => {
-								result = result[keyMain] ? result[keyMain] : "-";
+							keys.forEach((keyMain) => {
+								
+								if(showSerialNumber){
+								result = keyMain in result ? result[keyMain] : index+1;}
```
Comment (lakshita-yojak) [inline src/table/TableRow.tsx:124]: add space after paranthesis

**Review by lakshita-yojak — CHANGES_REQUESTED** [review]

**Comment by Tushar19varshney** [comment]
Please squash this PR.

**Comment by Tushar19varshney** [comment]
Great work @prateek-yojak 👍

**Pull request merged**